src/modules/processes.nix

```diff
@@ -53,7 +53,7 @@ in
       implementation = lib.mkOption {
         type = types.enum [ "honcho" "overmind" "process-compose" "hivemind" ];
         description = "The implementation used when performing ``devenv up``.";
-        default = "honcho";
+        default = "process-compose";
```

Comment:
🎉 

Please mention this change in the release docs, as even though this is a welcome change, it's a breaking one.

Reply:
While you are at it, I've completely migrated off of `devenv up` by using this one-liner `enterShell`:
https://github.com/schemamap/schemamap/blob/main/devenv.nix#L99-L101

The primary issue with `devenv up` at the moment is that it does not get evaluation cached, like nix-direnv'd devenv shell. Haven't tested yet with the Rust implementation, so feel free to correct me here.

The above automatically materializes the process-compose config yaml (into a gitignored file), which saves a ton of startup time compared to running `devenv up`.
Most of the config is already evaluated during `devenv shell`, so orcing the evaluation of the `processes` does not add significant overhead, while making day-to-day development/restarts way quicker.

I propose a boolean option to be turned on by default to add this one-liner to `enterShell`.
This way `devenv up` can be aliased to the `implementation` binary (without triggering Nix evaluation).

Reply:
https://github.com/cachix/devenv/pull/1005/files#diff-7104fdcadefcb5687bbdaeda5d6db909155e0dc3f83f8d97ace3302590aa098eR67

Also consider adding this new `is_strict = true;`, so there are less footguns.
https://f1bonacc1.github.io/process-compose/configuration/#strict-configuration-validation

Reply:
> While you are at it, I've completely migrated off of `devenv up` by using this one-liner `enterShell`: https://github.com/schemamap/schemamap/blob/main/devenv.nix#L99-L101
> 
> The primary issue with `devenv up` at the moment is that it does not get evaluation cached, like nix-direnv'd devenv shell. Haven't tested yet with the Rust implementation, so feel free to correct me here.
> 
> The above automatically materializes the process-compose config yaml (into a gitignored file), which saves a ton of startup time compared to running `devenv up`. Most of the config is already evaluated during `devenv shell`, so orcing the evaluation of the `processes` does not add significant overhead, while making day-to-day development/restarts way quicker.
> 
> I propose a boolean option to be turned on by default to add this one-liner to `enterShell`. This way `devenv up` can be aliased to the `implementation` binary (without triggering Nix evaluation).

That's pretty cool, I won't do that now since I'd like to release this finally, but we can revisit.